### PR TITLE
add Cymysql

### PR DESCRIPTION
**Pull request by rluria14**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by rluria14** [comment]
@benhuff @mwilson8 agree to be maintainers?

File: recipes/cymysql/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv "
+
+requirements:
+  host:
+    - pip
+    - python >=3.3
+  run:
+    - python >=3.3
```
Comment (marcelotrevisani) [inline recipes/cymysql/meta.yaml]: Don't you need ``cython``?

As the summary says "Python MySQL Driver using **Cython**"

Comment (marcelotrevisani) [reply recipes/cymysql/meta.yaml]: Actually, I read the ``setup.py`` and it seems you need ``cython`` on the ``host`` section to build some modules.

File: recipes/cymysql/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv "
+
+requirements:
+  host:
+    - pip
+    - python >=3.3
```
Comment (marcelotrevisani) [inline recipes/cymysql/meta.yaml]:
```suggestion
    - python >=3.3
    - cython
```

File: recipes/cymysql/meta.yaml (hunk 1 above)
Comment (rluria14) [reply recipes/cymysql/meta.yaml]: @marcelotrevisani thanks, cython has been added to host.

File: recipes/cymysql/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b2a20c6d5373a16590b75f149a7cb2215a6d3bb2fc856b1b75f437ccf1e716d4
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv "
+
+requirements:
```
Comment (marcelotrevisani) [inline recipes/cymysql/meta.yaml]: I think you will need to add the compilers here and remove ``noarch`` as well
```suggestion
requirements:
  build:
    - {{ compiler('cxx') }}
    - {{ compiler('c') }}
```

File: recipes/cymysql/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b2a20c6d5373a16590b75f149a7cb2215a6d3bb2fc856b1b75f437ccf1e716d4
+
+build:
+  noarch: python
```
Comment (marcelotrevisani) [inline recipes/cymysql/meta.yaml]:
```suggestion
```

**Comment by rluria14** [comment]
@marcelotrevisani any suggestions on fixing the win_64 build?

**Comment by marcelotrevisani** [comment]
@rluria14 , I did some modifications
If you are happy with that I will merge it

**Comment by rluria14** [comment]
> @rluria14 , I did some modifications
> If you are happy with that I will merge it

Perfect. Please go ahead and merge.

**Pull request merged**